packages/cadl-ranch-specs/http/payload/json-merge-patch/mockapi.ts

```diff
@@ -0,0 +1,76 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, json, MockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+function createMockApis(route: string, isUpdateRequest: boolean): MockApi {
+  const url = `/json-merge-patch/${route}`;
+  const expectedCreateBody = {
+    name: "Madge",
```

Comment:
What you expect user to send if they want to clear the dictionary?

Reply:
And a similar question is what you expect user to send if they want to add an item to the dictionary?

Reply:
No difference with setting other properties to `null`.